pkgs/by-name/ni/nist-feed/package.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, makeWrapper
+, bash
+, jq
+, killall
+, libnotify
+, cron
+}:
+
+stdenv.mkDerivation rec {
+  pname = "nist-feed";
+  version = "unstable-2024-01-20";
+
+  src = fetchFromGitHub {
+    owner = "D3vil0p3r";
+    repo = "NIST-Feed";
+    rev = "775bd871490b680784a1855cdc1d4958a83a7866";
+    hash = "sha256-OcVf766q7vELYkGOEzQMLS6zH8Nn96ibGP+6kizHN28=";
+  };
+
+  patches = [
+    ./cron.patch
+  ];
+
+  buildInputs = [ bash ];
+  nativeBuildInputs = [ makeWrapper ];
+
+  postPatch = ''
+    substituteInPlace nist-feed \
+      --replace "/usr/local/bin/nist-feed" $out/bin/nist-feed
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm755 nist-feed -D $out/bin/nist-feed
+    wrapProgram "$out/bin/nist-feed" \
+      --prefix PATH : "$out/bin:${lib.makeBinPath [ jq killall libnotify ]}"
```

Comment:
I'm wondering whether the `curl` and `busybox` from the service should be added here? Do you know what this program needs to run fine @D3vil0p3r?

Reply:
Yes for `curl`, for `busybox` mmm, not sure @h7x4 @octodi

Reply:
I suppose that would be for `cp`, `mv`, `rm`, etc. I think coreutils is pretty much guaranteed (although I might be wrong), so maybe we could try removing `busybox`.

Reply:
Oh yes. You're right

Reply:
@h7x4 systemd popped me with unavailability of `curl` and `pkill` therefore I aligned with that solution, Though I have to test in case it works with wrapping them with the program